### PR TITLE
Remove unnecessary margin at Home landing page #6454

### PR DESCRIPTION
**Pull request by gautampanday**

### CODE REVIEW

**Comment by gautampanday** [comment]
home tryton world s most stable opensource erp system

File: theme/templates/base.html [hunk 1]
```diff
@@ -94,85 +94,85 @@
   {% block request_demo %}
   <section class="know-more pull-left pad-all20">
```
Comment (GauravButola) [inline theme/templates/base.html]: You can remove pad-all20 and just add padding to the already available class.

File: theme/static/css/style.css [hunk 2]
```diff
@@ -119,38 +129,6 @@ a:hover {
   border-right:1px solid #ddd;
 }
 
-.mar_top2 {
```
Comment (GauravButola) [inline theme/static/css/style.css:122]: Also remove all these classes from HTML, wherever they are being used.

File: theme/static/css/style.css [hunk 3]
```diff
@@ -223,11 +201,6 @@ a:hover {
   color: #999;
 }
 
-.footer .footer_headings {
-  color: #fff;
-  font-weight: bold;
-}
-
 .footer .contact_address {
```
Comment (GauravButola) [inline theme/static/css/style.css]: Use consistent class naming convention ie. use hyphens to separate words in classes.

**Comment by gautampanday** [comment]
foundation tryton world s most stable opensource erp system

**Comment by gautampanday** [comment]
tryton world s most stable opensource erp system

**Comment by gautampanday** [comment]
app tryton world s most stable opensource erp system

**Comment by gautampanday** [comment]
ecommerce tryton world s most stable opensource erp system

**Comment by gautampanday** [comment]
accounting app tryton world s most stable opensource erp system

**Comment by gautampanday** [comment]
inventory management tryton world s most stable opensource erp system

**Comment by gautampanday** [comment]
community world s most stable opensource erp system

**Comment by gautampanday** [comment]
blog world s most stable opensource erp system

File: theme/templates/accounts.html [hunk 4]
```diff
@@ -23,11 +22,11 @@ <h1 class="white-text-color"><strong>{% trans %}ACCOUNTING{% endtrans %}</strong
     </div>
   </div>
 </section>
-<div class="mar_top5 clearfix"></div>
+<div class="clearfix"></div>
```
Comment (tarunbhardwaj) [inline theme/templates/accounts.html]: I doubt you really need clearfix here, as all sections will be take full width. and clearfixed

**Pull request merged**